lib/exceptions.h

```diff
@@ -46,18 +46,26 @@ class MYSQLPP_EXPORT Exception : public std::exception
 {
 public:
 	/// \brief Create exception object as copy of another
+#if __cplusplus >= 201103L
+    Exception(const Exception&) MAY_THROW() = default;
```

Comment:
The exchange of trivial ctors with "= default" makes the code shorter by making it longer. The new style is justified only if you're able to throw away portability to older compilers, which we can't in MySQL++.